tests/sanity_check.sh

```diff
@@ -8,7 +8,7 @@ fail_msg="*****************************Sanity check for quda failed!************
 function basic_sanity_check {
```

Comment:
Not sure this file should be add here. If we want something like this we should add it to ctest.

Reply:
To be fair, this was a bit of a `sed` adventure since I renamed `gauge_force_test` to `gauge_path_test`. I'm down with removing this file since it seems a bit legacy (and I've never used it), but I think it should be left for a separate PR. Any objections?